### DISCUSSION TITLE
我觉得添加一个新旧版本ui切换按钮

### DISCUSSION THREAD

**Discussion by hedan2** [lollipopkit/flutter_server_box#146] [Ideas]
有习惯新版本的用户，也有习惯旧版本ui的用户，添加一个切换按钮可以使部分用户用起来更加方便快捷，让用户对该应用更加喜欢

**Comment by lollipopkit** [comment]
已在 515 中实现，请等待发布更新，并根据 [Wiki](https://github.com/lollipopkit/flutter_server_box/wiki/主页#以-json-编辑设置) 来设置
